documentation/api/query/v4/aggregate-event-counts.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PuppetDB 3.2 » API » v4 » Querying Aggregate Event Counts"
+title: "PuppetDB 3.2: Aggregate event counts endpoint"
```

Comment:
If this new format is somehow the new standard, then fine - I just think sometimes its easy to get lost in these docs, if I look at the Puppet docs, there is a feeling of reduced taxonomy over having a breadcrumb trail. Hohum. Compare it to the mcollective and pdb docs for example, where there is a breadcrumb trail somewhere on the page.

If I was honest with myself, its the breadcrumb trail concept that I'd miss. And right now we have a breadcrumb above the TOC, and in the title which does seem overkill.

In a way, I wish the breadcrumb was driven by the TOC, and the title could be whatever we wanted. Perhaps that is a better approach ... you can keep your short titles and we let the code do the breadcrumb based on the TOC location perhaps.

Also yes, the inconsistency with other pages and other projects like mcollective and puppet should be fixed, no matter what we do. I've been scanning other projects and it seems like we have 2-3 different ways of doing this. Sometimes the breadcrumb concept is just done with colons, sometimes with those weird arrows ... sometimes there is no breadcrumb.